attest/net/Cargo.toml

```diff
@@ -25,7 +25,7 @@ mbedtls = { git = "https://github.com/mobilecoinofficial/rust-mbedtls.git", tag
 cfg-if = "0.1"
 failure = { version = "0.1.5", features = ["derive"] }
 pem = "0.6"
-reqwest = { version = "0.10" , default-features = false, features = ["rustls-tls"] }
+reqwest = { version = "0.10" , default-features = false, features = ["rustls-tls", "gzip"] }
```

Comment:
This stuff was breaking during `make dist` - I have a ticket to see why this isn't exercised as part of CI.